src/components/home/Chart/IncidentChart.js

```diff
@@ -108,39 +111,54 @@ const IncidentChart = ({ color, chart_data, state, isFirstLoadData }) => {
           <div className="drop-down" />
         </>
       ) : null}
-      <ResponsiveContainer >
+
+      <ResponsiveContainer>
       <ComposedChart 
-            height={300}
-            width={794} 
+            height={382}
+            width={1000}
```

Comment:
Hardcode the width make the chart not responsive to window size change. Can we follow the main branch, only set the height?

Reply:
Thank you for your attention the chart is not responsive to window size changes, I will fix it followed by the main branch.